hack/sigstore-setup/sigstore-setup.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+#
+# Copyright 2021 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is copied from sigstore/sigstore
+# replacing docker-compose with podman-compose
+# See https://github.com/sigstore/sigstore/blob/main/test/e2e/e2e-test.sh
+
+set -ex
+
+cleanup() {
+  echo "cleanup"
+  $COMPOSE_CMD down
+}
+
+run=$1
+if [ $run = "cleanup" ]
+then
+    cleanup
+else
+  trap cleanup ERR
+
+  export VAULT_TOKEN=testtoken
+  export VAULT_ADDR=http://localhost:8200/
+
+  # The following env vars are required only when running sigstore e2es
+  # These are not required for sigstore setup
+  export AWS_ACCESS_KEY_ID=test
+  export AWS_SECRET_ACCESS_KEY=test
+  export AWS_REGION=us-east-1
+  export AWS_ENDPOINT=localhost:4566
+  export AWS_TLS_INSECURE_SKIP_VERIFY=1
+  export OIDC_ISSUER=http://127.0.0.1:5556/auth
```

Comment:
Should we export the IPs in the Makefile to make them accessible to all tests?

Reply:
right, yup - updated/added to Makefile - also the OIDC vars are not necessary, removed those